.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug",
+            "type": "go",
+            "request": "launch",
+            "mode": "debug",
+            "remotePath": "",
+            "port": 2345,
+            "host": "127.0.0.1",
+            "program": "${workspaceRoot}",
+            "envFile": "${workspaceFolder}/.env.local",
+            "args": [],
+            "showLog": true
+        },
+        {
+            "name": "Integration tests - controller",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceFolder}/controllers",
+        },
+        {
+            "name": "Unit tests - scheduler",
+            "type": "go",
+            "request": "launch",
+            "mode": "test",
+            "program": "${workspaceFolder}/internal/scheduler",
+        }
+        {
+            "name": "Unit tests - utils",
```

Comment:
Go extension is a bit annoying, it doesn't allow you to specify recursive debugging, so we either have to set it like this or change the target program as needed in case that we want to debug some unit tests.